datarepo-clienttests/src/main/java/scripts/testscripts/SnapshotScaleDelete.java

```diff
@@ -0,0 +1,124 @@
+package scripts.testscripts;
+
+import bio.terra.datarepo.api.DataRepositoryServiceApi;
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.DeleteResponseModel;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotModel;
+import bio.terra.datarepo.model.SnapshotSummaryModel;
+import com.google.cloud.storage.BlobId;
+import common.utils.StorageUtils;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.config.TestUserSpecification;
+import scripts.testscripts.baseclasses.SimpleDataset;
+import scripts.utils.DataRepoUtils;
+
+public class SnapshotScaleDelete extends SimpleDataset {
+  private static final Logger logger = LoggerFactory.getLogger(SnapshotScaleDelete.class);
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public SnapshotScaleDelete() {
+    super();
+  }
+
+  private SnapshotModel snapshotModel;
+
+  private static List<BlobId> scratchFiles = new ArrayList<>();
+  private String dirObjectId;
+
+  private int NUM_SNAPSHOTS = 1;
```

Comment:
Since this is not a static final, it shouldn't be in all caps.
```suggestion
  private int numSnapshots = 1;
```